monitor/main.go

```diff
@@ -42,6 +42,7 @@ func Start(ctx context.Context, cfg *Config) error {
 
 	// Todo: Make this configurable
 	datadir := filepath.Join(homeDir(), ".eksportisto")
+	os.MkdirAll(datadir, os.ModePerm)
```

Comment:
What happens if the dir already exists? I'd personally like to see the datadir to just be configuratabble like geth or rosetta

Reply:
Nothing happens if it already exists. Sure configurable is better, will do a command line flag for now